src/plugins/table/table-utils.js

```diff
@@ -36,9 +40,22 @@ export const DESIGNER_CELL_ACTION = {
   connectToRowAfter: 'connect-to-row-after',
   connectToColumnBefore: 'connect-to-column-before',
   connectToColumnAfter: 'connect-to-column-after',
-  disconnectCell: 'disconnect-cell'
+  disconnectAllCells: 'disconnect-all-cells',
+  disconnectCell: 'disconnect-cell',
+  setVerticalAlignmentToTop: 'set-vertical-alignment-to-top',
+  setVerticalAlignmentToMiddle: 'set-vertical-alignment-to-middle',
+  setVerticalAlignmentToBottom: 'set-vertical-alignment-to-bottom',
+  setHorizontalAlignmentToLeft: 'set-horizontal-alignment-to-left',
+  setHorizontalAlignmentToCenter: 'set-horizontal-alignment-to-center',
+  setHorizontalAlignmentToRight: 'set-horizontal-alignment-to-right'
 };
 
+export function isCellHit(cell, rowIndex, columnIndex) {
```

Comment:
Small thing, how about `isCellAtCoordinates`? Took me a bit to get the `hit` part and then O directly thought of the battleships game 😂

Reply:
Myeah, it comes from "hit testing" in graphics development when checking if an object is "hit" by a location. I wanted to avoid anything with "coordinates" in it, because the cell is also "hit" when e.g. `rowIndex` is `-1` (meaning: all cells that touch the column at `columnIndex`, no matter which row). So, as `rowIndex` and `columnIndex` do not necessarily constitute a real cell coordinate, I was searching for more abstract things like `isCellConcerned`, but as "concerned" sounded too human to me, I went with "hit". Would `isCellAffected` sound better to you as an alternative?

Reply:
Ok I see, hadn't realized they are not quite coordinates 🤔 
`isCellAffected` sounds good though.

Reply:
Ok, I actually like it better, too (haven't thought of "affected" initially). Changed!